### PR TITLE
Task_#53365--Added Header and Card components

### PR DESCRIPTION
**Pull request by CosminaX**

### CODE REVIEW

File: src/Components/Card/Card.js [hunk 1]
```diff
@@ -1,7 +1,59 @@
-import React from "react";
+import React, { useState } from "react";
+import "./card.scss";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
```
Comment (BiancaDianaAlexiuc) [inline src/Components/Card/Card.js:6]: It's considered a best practice to have the imports from libraries at top of the page and the imports from custom components after that

File: src/Components/Header/Header.js [hunk 2]
```diff
@@ -1,7 +1,107 @@
...
+import "./header.scss";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
+import Button from "@mui/material/Button";
+import Menu from "@mui/material/Menu";
+import fav from "./fav.png";
+import myacc from "./myacc.png";
+import logout from "./logout-icon.png";
+import messages from "./messages-icon.png";
```
Comment (petricafinaga) [inline src/Components/Header/Header.js:11]: All the images should be moved into some assets folder and used from there

**Review by BiancaDianaAlexiuc — APPROVED** [review]

File: src/Components/Card/Card.js [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+        <div className="like" onClick={() => handleLike()}>
+          {favorite === false ? (
+            <Like className="like-icon" />
+          ) : (
+            <Liked className="like-icon" />
+          )}
+        </div>
+
+        <CardMedia className="card-img" component="img" image={casa1} />
+        <CardContent className="card-descriprion">
```
Comment (petricafinaga) [inline src/Components/Card/Card.js:31]: I think you have misspelled the class name 😄

File: src/pages/homepage/_slick-theme.scss [hunk 4]
```diff
@@ -0,0 +1,185 @@
+@charset 'UTF-8';
+// /* Slider */
+// .slick-loading .slick-list {
+//   background: #fff url("./ajax-loader.gif") center center no-repeat;
+// }
+
+// /* Icons */
+// @font-face {
```
Comment (petricafinaga) [inline src/pages/homepage/_slick-theme.scss:8]: If you think you won't need this css in the future it's better to remove it 😸

**Review by petricafinaga — APPROVED** [review]

**Pull request merged**